src/contrib/cluster/Akka.Cluster.Sharding/ShardRegion.cs

```diff
@@ -294,21 +308,18 @@ protected object RegistrationMessage
             }
         }
 
-        /// <summary>
-        /// TBD
-        /// </summary>
+        /// <inheritdoc cref="ActorBase.PreStart"/>
```

Comment:
Docfx supports only plain `<inheritdoc>` and it could inherit from the base class or from interface. But it could not inherit the documentation from the custom elements